db/migrate/20250926060748_add_index_to_contacts_additional_attributes_chatwoot_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToContactsAdditionalAttributesChatwootId < ActiveRecord::Migration[7.1]
+  def change
+    add_index :contacts, "(additional_attributes->>'chatwoot_id')", name: 'index_contacts_on_chatwoot_id'
```

Comment:
Esse where reduz o tamanho do índice, ignorando todos os contatos que não tem esse campo

```suggestion
    add_index :contacts,
              "(additional_attributes->'chatwoot_id')",
              name: 'index_contacts_on_chatwoot_id',
              where: "additional_attributes->'chatwoot_id' IS NOT NULL"
```